CONTRIBUTING.md

```diff
@@ -44,6 +44,25 @@ Build logs are written to stdout and artifacts written to the current directory.
 When new versions of the packages are released, they should be included in [CHaP](https://github.com/input-output-hk/cardano-haskell-packages).
 See the CHaP README for [instructions](https://github.com/input-output-hk/cardano-haskell-packages#-from-github).
 
+## Release Branches
+
+When needed we use release branches: `release/cardano-node-*`, e.g.
+`release/cardano-node-1.35.x` (which ought to be simply named as
+`release/cardano-node-1.35`).
+
+Note that `CHaP` allows us to make releases of packages independently of each
+other (especailly non breaking changes), so there might be other release
+branches, e.g.  `release/network-mux-*`.  They MUST follow the following
+nameing convention: `release/${package-name}-${version}`.
+
+All commits in the release branch MUST be cherry-picked from `master`.  Each
+time a new commits are added to a release branch one MUST open a pull request
```

Comment:
```suggestion
time new commits are added to a release branch one MUST open a pull request
```

Reply:
I don't really understand this. I think you mean:

"once new commits from master want to be added to a release branch, one must
1. create a branch on top of the release branch
2. cherry pick commits from master on that new branch
3. open a pull request from that branch into the release branch"

I didn't put the above as suggestion because I might be wrong in my understanding

Reply:
That's exactly what I was thinking about.

Reply:
I updated this section.  I also added that one should mention in the PR description from which original PRs the cherry-picked commits are coming from.

Should we also say that preferably one should cherry-pick all commits from a single PR?
This can save problems when later one needs to cherry-pick more commits from other PRs which have subtle dependencies on un cherry-picked changes.